vlm/src/main/scala/geotrellis/contrib/vlm/gdal/GDALResampleRasterSource.scala

```diff
@@ -29,7 +29,8 @@ case class GDALResampleRasterSource(
   uri: String,
   resampleGrid: ResampleGrid,
   method: ResampleMethod = NearestNeighbor,
-  baseWarpList: List[GDALWarpOptions] = Nil
+  baseWarpList: List[GDALWarpOptions] = Nil,
+  alignTargetPixels: Boolean = true
```

Comment:
In order to not perpetuate mistakes of our fathers this `GDALResampleRasterSource` needs a ScalaDoc that explains this option in particular.